Scripts/Modules/SnappingModule/SnappingModule.cs

```diff
@@ -532,7 +532,11 @@ void AddToIgnoreList(GameObject[] objects)
 
 			for (int i = 0; i < objects.Length; i++)
 			{
-				m_CombinedIgnoreList.Add(objects[i]);
+				var renderers = objects[i].GetComponentsInChildren<Renderer>();
```

Comment:
Since the snapping module just went in with this release would this be a bug that would go out with 0.0.4?